### PR TITLE
Enh/1/assignment2

### LINKED ISSUE #1: Issue_Assignment_2
- Create a new branch **(enh/issue-id/short-msg)** and copy the code and commit as -is including the README.
- Add a  **.gitignore file to the repository to avoid committing unnecessaey files like **pyc** files and the **csv** files.
- Use Conda to set up a python development environment named **mle-dev** run the script . export the environment file as **env.yml** and commit it to the repository.
- Update the **README** file with instructions on how to run the code .
- Create a ** Pull Request (PR)** . Link the PR to the **GitHub issue** to ensure the issue is closed automatically when the PR is mergerd.
- Add the **facililators** for session as collaborators on the **mle-training** repository and request a **PR**.
- Merge the PR to **master** branch and give it **tag** **'v0.1'** and push that to the remote.

### PR DESCRIPTION
**Pull request by songarajeet**
closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
...
+ - Random Forest
+
+## Steps performed
+ - We prepare and clean the data. We check and impute for missing values.
+ - Features are generated and the variables are checked for correlation.
+ - Multiple sampling techinuqies are evaluated. The data set is split into train and test.
+ - All the above said modelling techniques are tried and evaluated. The final metric used to evaluate is mean squared error.
+
+## To excute the script
+python < scriptname.py >
```
Comment (sharma-sandy) [inline README.md:18]: Syntax highlighting can be used  for example: `python nonstandardcode.py`

File: env.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - pip=21.2.4=py39haa95532_0
+  - python=3.9.12=h6244533_0
+  - setuptools=61.2.0=py39haa95532_0
+  - sqlite=3.38.3=h2bbff1b_0
+  - tzdata=2022a=hda174b7_0
+  - vc=14.2=h21ff451_1
+  - vs2015_runtime=14.27.29016=h5e58377_2
+  - wheel=0.37.1=pyhd3eb1b0_0
+  - wincertstore=0.2=py39haa95532_2
+prefix: C:\Users\jitendra.kumar\Anaconda3\envs\mle-dev
```
Comment (sharma-sandy) [inline env.yml:17]: remove prefix as it is a system specific path

**Review by sharma-sandy — CHANGES_REQUESTED** [review]
Do the suggested minor changes and re-raise request

File: env.yml (hunk 2 above)
Comment (songarajeet) [reply env.yml]: okay

**Review by sharma-sandy — APPROVED** [review]
You could have used the code highlighting part. But approving the PR for now. Please make necessary changes in the next PR

**Pull request merged**